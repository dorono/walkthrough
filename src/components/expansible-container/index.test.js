import React from 'react';
import {shallow} from 'enzyme';
import ExpansibleContainer from './index';

describe('ExpansibleContainer', () => {
    it('should render without errors', () => {
        shallow(<ExpansibleContainer />);
    });

    it('should match snapshot', () => {
        expect(shallow(<ExpansibleContainer />)).toMatchSnapshot();
    });
});
