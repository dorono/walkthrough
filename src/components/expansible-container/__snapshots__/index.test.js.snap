// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpansibleContainer should match snapshot 1`] = `
<SizeMeRenderer(ExpansibleContainer)
  disablePlaceholder={false}
  explicitRef={[Function]}
  size={
    Object {
      "height": undefined,
      "position": undefined,
      "width": undefined,
    }
  }
/>
`;
